src/clr-angular/accordion/_accordion.clarity.scss

```diff
@@ -0,0 +1,221 @@
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('accordion.clarity') {
+  .clr-accordion {
+    display: block;
+    counter-reset: accordion;
+    margin-bottom: 24px;
+  }
+
+  .clr-accordion-panel {
+    display: block;
+  }
+
+  .clr-accordion-header {
+    color: $clr-accordion-text-color;
+    border: 1px solid $clr-accordion-border-color;
+    box-shadow: inset $clr-accordion-border-left-width 0 0 $clr-accordion-border-left-color;
+    background: $clr-accordion-header-background-color;
+    transition: all 200ms ease-in-out;
+    border-bottom: 0;
+    display: flex;
+    flex-flow: row wrap;
+    width: 100%;
+    font-size: 0.65rem;
+    text-align: left;
+    @include focus-within();
+
+    &:hover {
+      background-color: $clr-accordion-header-hover-background-color;
+    }
+
+    @media (min-width: $clr-accordion-responsive-breakpoint) {
+      flex-flow: row;
+    }
+  }
+
+  .clr-accordion-panel:last-child {
+    .clr-accordion-content,
+    .clr-accordion-header {
+      border-bottom: 1px solid $clr-accordion-border-color;
+    }
+  }
+
+  .clr-accordion-number {
+    padding: 0 18px;
+    display: none;
+
+    &::before {
+      content: counter(accordion) '.';
+      counter-increment: accordion;
+    }
+  }
+
+  .clr-accordion-header-button {
+    width: 100%;
+    border: 0;
+    padding: $clr-accordion-padding;
+    background: transparent;
+    text-align: left;
+    cursor: pointer;
+    color: $clr-accordion-text-color;
+
+    @media (min-width: $clr-accordion-responsive-breakpoint) {
+      max-width: 280px;
+    }
+  }
+
+  button.clr-accordion-header-button {
+    outline: none; // parent .clr-accordion-header element contains focus style
+  }
+
+  .clr-accordion-status {
+    width: 35px;
+    display: inline-block;
+    vertical-align: top;
+  }
+
+  .clr-accordion-title {
+    width: calc(100% - 35px);
+    max-width: 280px;
+    display: inline-block;
+  }
+
+  .clr-accordion-description {
+    padding: $clr-accordion-padding;
+    padding-top: 0;
+    width: 100%;
+    max-width: 650px;
+
+    @media (min-width: $clr-accordion-responsive-breakpoint) {
+      padding-top: $clr-accordion-padding;
+    }
+  }
+
+  .clr-accordion-content {
+    background: $clr-accordion-content-background-color;
+    border: 1px solid $clr-accordion-border-color;
+    border-bottom: 0;
+    overflow: hidden;
+    display: block;
+    padding: 20px;
+    display: none;
+
+    .clr-form {
+      padding: 0;
+    }
+
+    &.ng-trigger {
+      padding: 0; // inner element/padding to prevent extra hight in ng animation
+    }
+
+    @media (min-width: $clr-accordion-responsive-breakpoint) {
+      padding: 20px 54px;
+    }
+  }
+
+  .clr-accordion-inner-content {
+    padding: 20px;
+
+    @media (min-width: $clr-accordion-responsive-breakpoint) {
+      padding: 20px 54px;
```

Comment:
I think we need to turn these px values to rem. Also, looks like some values are not in accord with our baseline, ie: 20px, 13px, etc.,? So can we turn 35px to 1.5rem and 20px to 0.75rem, etc?

Reply:
Updated to use rems